src/components/layout/modal/browser_update_alert_modal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react'
+import { ModalRefType } from 'components/elements/modal'
+import { Localize } from 'components/localization'
+import { Modal, Header } from 'components/elements'
+import { useUserBrowser } from 'components/hooks/use-user-browser'
+import { CookieStorage } from 'common/storage'
+import { cookie_key, browsers_minimum_required_version } from 'common/constants'
+
+const BrowserUpdateAlertModal = () => {
+    const modalRef = React.useRef<ModalRefType>()
+    const { is_outdated } = useUserBrowser(browsers_minimum_required_version)
```

Comment:
do we have a situation where we need to pass another config for this? If not can we keep `browsers_minimum_required_version` inside the `useUserBrowser` so we don't need to pass ot like this?

Reply:
Right now we don't have such situation.
At first I kept `browsers_minimum_required_version` inside the `useUserBrowser` but then we decided to change it in a way that everyone be able to pass any config that they want. I can make the param optional and use `browsers_minimum_required_version` inside the hook as the default value for the param. 
what do you think @prince-deriv ?
@ali-hosseini-fs can you give your opinion on this?